db/migrate/20221101193016_create_comments.rb

```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.references :user, null: false, foreign_key: true
```

Comment:
Please add `author_id` as a foreign key instead of `user_id` as it's specified in the requirements.